README.md

```diff
@@ -265,34 +265,34 @@ Each `qna.yaml` file requires a minimum of five question-answer pairs. The `qna.
 
 ### Knowledge: YAML examples
 ```yaml
-task_description: 'Teach the model the results of the 2024 oscars'
+task_description: 'Teach the model the results of the 2024 Oscars'
 created_by: juliadenham
 domain: pop_culture
 seed_examples:
- - question: When did the 2024 oscars happen?
+ - question: When did the 2024 Oscars happen?
    answer: |
-     The 2024 oscars were held on March 10, 2024.
- - question: What film had the most oscar nominations in 2024?
+     The 2024 Oscars were held on March 10, 2024.
+ - question: What film had the most Oscar nominations in 2024?
    answer: |
-     Oppenheimer had 13 oscar nominations.
- - question: Who presented the awards for Best Original Screenplay and Best Adapted Screenplay?
+     Oppenheimer had 13 Oscar nominations.
+ - question: Who presented the 2024 Oscar for Best Original Screenplay and Best Adapted Screenplay?
    answer: |
      Octavia Spencer.
```

Comment:
Another suggestion here, we need to give proper, full statements and avoid one word/short answers.
```suggestion
     Octavia Spencer hosted the 2024 Oscars.
```